_episodes/01-intro.md

```diff
@@ -59,98 +59,61 @@ loops back and waits for you to enter another command.
 
 
 The Shell is a program which runs other programs rather than doing calculations itself.
-Those programs can be as complicated as a climate modeling software and as simple as a
-program that creates a new folder/directory. The simple programs which are used to perform
+Those programs can be as complicated as climate modeling software and as simple as a
+program that creates a new directory. The simple programs which are used to perform
 stand alone tasks are usually refered to as commands.
 The most popular Unix shell is Bash, (the Bourne Again SHell --- so-called because
 it's derived from a shell written by Stephen Bourne).
 Bash is the default shell on most modern implementations of Unix
 and in most packages that provide Unix-like tools for Windows.
 
-### What does it look like?
```

Comment:
Not a big deal either way, but I kind of liked this section heading

Reply:
So did I, but given we're no longer jumping in at the deep end, the heading seems a bit OTT:
## What does it look like? 
A prompt.